.github/workflows/build-and-push.yaml

```diff
@@ -0,0 +1,169 @@
+            environment_name="prod"
+            credentials_json='${{ secrets.PIPELINE_EPO_PROD_PROJECT }}'
+          elif ${{ contains(github.ref_name, 'main') || contains(github.base_ref, 'main') }}; then
+            environment_name="int"
+            credentials_json='${{ secrets.SKYVIEWER_INT_SERVICE_ACCOUNT }}'
+          else
+            environment_name="dev"
+            credentials_json='${{ secrets.DEV_SA_KEY }}'
+          fi
+          echo environment_name=$environment_name >> "$GITHUB_OUTPUT"
+          echo credentials_json=$credentials_json >> "$GITHUB_OUTPUT"
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5.7.0
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            gcr.io/skyviewer/orbitviewer-api,enable=${{ steps.deploy-info.outputs.environment_name == 'dev' }}
+            gcr.io/edc-int-6c5e/orbitviewer-api,enable=${{ steps.deploy-info.outputs.environment_name == 'int' }}
+            gcr.io/edc-prod-eef0/orbitviewer-api,enable=${{ steps.deploy-info.outputs.environment_name == 'prod' }}
+          flavor: |
+            latest=${{ github.event_name == 'push'}}
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=schedule
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+            type=sha
+      -
+        name: Parse deploy tag
+        id: deploy-tag
+        run: |
+          # Parse container image tag to deploy
+          full_tag=$(echo "$DOCKER_METADATA_OUTPUT_JSON" | jq -r '.tags[] | limit(1; select(. | test(":sha-|:v.")))')
+          echo "Will use tag \"$full_tag\" for deployment."
+          echo image_tag=$(echo "$full_tag" | cut -f2 -d:) >> "$GITHUB_OUTPUT"
+          echo image_name=$(echo "$full_tag" | cut -f1 -d:) >> "$GITHUB_OUTPUT"
+          echo full_tag=$full_tag >> "$GITHUB_OUTPUT"
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v3.6.0
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3.11.1
+      - 
+        name: Login to GCP
+        uses: 'google-github-actions/auth@v2.1.11'
+        with:
+          credentials_json: ${{ steps.deploy-info.outputs.credentials_json }}
+      - 
+        name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v2.1.5'
+      - 
+        run: gcloud --quiet auth configure-docker
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6.18.0
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-to: |
+            type=gha
+          cache-from: |
+            type=gha
+          build-args: |
+            BASE_TAG=k8s
```

Comment:
@jtrouth I noticed our other repos still use this tag, but it's no longer necessary correct? We should just be able to use `latest`?